src/ir_Mitsubishi.h

```diff
@@ -36,6 +36,7 @@
 //   Brand: Mitsubishi Electric,  Model: MSZ-ZW4017S A/C (MITSUBISHI_AC)
 //   Brand: Mitsubishi Electric,  Model: MSZ-FHnnVE A/C (MITSUBISHI_AC)
 //   Brand: Mitsubishi Electric,  Model: RH151 remote (MITSUBISHI_AC)
+//   Brand: Mitsubishi Electric,  Model: PAR-FL32MA/PAR-FA32MA remote (MITSUBISHI136)
```

Comment:
`src/ir_Mitsubishi.h:39:  Lines should be <= 80 characters long  [whitespace/line_length] [2]`
Please split this and have one remote per line, a.k.a.

While at it, which A/C unit is this?

```
//   Brand: Mitsubishi Electric,  Model: PAR-FL32MA remote (MITSUBISHI136)
//   Brand: Mitsubishi Electric,  Model: PAR-FA32MA remote (MITSUBISHI136)
```